vitrina/datasets/migrations/0016_populate_dataset_temporal_dates.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 3.2.25 on 2025-08-18 09:39
+from datetime import datetime
+
+from django.apps.registry import Apps
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.models import Min, Max
+
+
+def populate_dataset_temporal_dates(
+    apps: Apps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    Dataset = apps.get_model("vitrina_datasets", "Dataset")
+    Distribution = apps.get_model("vitrina_resources", "DatasetDistribution")
+    DCATResourceSubclass = apps.get_model("vitrina_datasets", "DCATResourceSubclass")
+
+    dataset_subclass = DCATResourceSubclass.objects.get(name="dataset")
+    datasets = Dataset.objects.filter(subclass=dataset_subclass)
+    stats = (
```

Comment:
Unless I am missing something:
```suggestion
    distributions = (
```